bookShop/src/app/modules/components/book/book-component/book-component.component.ts

```diff
@@ -7,18 +16,17 @@ import { IBook, Category } from '../../../../shared/models/BookModel';
   styleUrls: ['./book-component.component.scss'],
 })
 export class BookComponentComponent implements OnInit {
-
-  @Input() bookData: IBook[] | undefined;
+  bookData: Observable<IBook[]>;
 
   @Output() buyEvent = new EventEmitter<IBook>();
-  constructor() {}
-
-  ngOnInit(): void {
+  constructor(private booksService: BooksService) {
+    this.bookData = this.booksService.getBooks();
```

Comment:
Add $ in the end of ` this.bookData` because it's Observable.

Reply:
Corrected